Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Core/EBusEventHandler.cpp

```diff
@@ -602,14 +602,24 @@ namespace ScriptCanvas
                 }
             }
 
-            void EBusEventHandler::OnWriteEnd()
+            void EBusEventHandler::OnDeserialize()
             {
                 AZStd::lock_guard<AZStd::recursive_mutex> lock(m_mutex);
                 if (!m_ebus)
                 {
                     CreateHandler(m_ebusName);
                 }
+
+                /// \note Call super() anti-pattern https://en.wikipedia.org/wiki/Call_super
```

Comment:
I'm not sure it's permissible to add urls, no need to call this out either

Reply:
x

Reply:
Links to generally available information such as Wikipedia articles about programming patterns and algorithms or StackOverflow pages with referenced solution are fine. (It's actually considered a good thing so the people sharing information get credit for it.)